object-manipulation-3/main.js

```diff
@@ -0,0 +1,125 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var players = createPlayers(4);
+playGame(players, 2);
+
+function playGame(playerList, numCards) {
+  var deck = createDeck();
+  deck = _.shuffle(deck);
+  for (var i = 0; i < playerList.length; i++) {
+    playerList[i].hand = drawCards(deck, numCards);
+  }
+  console.log('Player\'s Hands:', playerList);
+  getScores(playerList);
+  var winner = getWinner(playerList);
+  var newPlayerList = checkForTies(playerList, winner);
+  if (newPlayerList.length > 1) {
+    console.log('A tie has occurred so a tie-breaker game will run');
+    playGame(newPlayerList, 2);
+  } else {
+    console.log('The winner is', winner.name);
+  }
+}
+
+function createPlayers(numPlayers) {
+  var playerArray = [];
+  for (var i = 1; i <= numPlayers; i++) {
+    playerArray.push({ name: ('Player ' + i), hand: null });
+  }
+  return playerArray;
+}
+
+function createDeck() {
+  var deck = [];
+  var suits = ['clubs', 'diamonds', 'hearts', 'spades'];
+  var ranks = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K'];
+  for (var i = 0; i < suits.length; i++) {
+    for (var n = 0; n < ranks.length; n++) {
+      deck.push({ rank: ranks[n], suit: suits[i] });
+    }
+  }
+  return deck;
+}
+
+function drawCards(deck, numCards) {
+  var drawnCards = [];
+  for (var i = 0; i < numCards; i++) {
+    var card = deck[Math.floor(Math.random() * deck.length)];
+    drawnCards.push(card);
+    deck.splice(deck.indexOf(card), 1);
+  }
+  return drawnCards;
+}
+
+function getScores(players) {
+  for (var i = 0; i < players.length; i++) {
+    var score = 0;
+    for (var n = 0; n < players[i].hand.length; n++) {
+      if (players[i].hand[n].rank === 'A') score += 11;
+      else if (players[i].hand[n].rank === 'J' || players[i].hand[n].rank === 'Q' || players[i].hand[n].rank === 'K') score += 10;
+      else {
+        score += parseInt(players[i].hand[n].rank);
+      }
+    }
+    players[i].score = score;
+  }
+}
+
+function getWinner(players) {
+  var winner = players[0];
+  for (var i = 0; i < players.length; i++) {
```

Comment:
This can start at `1` since you have already saved the player at `0` as the default winner to start